<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="minimumSize">
          <size>
           <width>712</width>
           <height>161</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>161</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QGroupBox" name="groupBox">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>461</width>
            <height>131</height>
           </rect>
          </property>
          <property name="title">
           <string>Parametros de Execução</string>
          </property>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="geometry">
            <rect>
             <x>210</x>
             <y>10</y>
             <width>241</width>
             <height>111</height>
            </rect>
           </property>
           <property name="title">
            <string>Round com os sistemas. </string>
           </property>
           <widget class="QCheckBox" name="check_sys_contabil">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>111</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Contabilidade </string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QCheckBox" name="check_sys_compras">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>40</y>
              <width>111</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Compras</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QCheckBox" name="check_sys_licitacao">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>60</y>
              <width>111</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Licitação</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QCheckBox" name="check_sys_arh">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>80</y>
              <width>111</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>ARH</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QCheckBox" name="check_sys_solicitaoweb">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>60</y>
              <width>111</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Solicitação Web</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QCheckBox" name="check_sys_gecom">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>80</y>
              <width>111</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Gecom</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QCheckBox" name="check_sys_siart">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>20</y>
              <width>111</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Siart</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QCheckBox" name="check_sys_docfacil">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>40</y>
              <width>111</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>DocFácil</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </widget>
          <widget class="QCheckBox" name="check_param_browser">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>70</y>
             <width>111</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Ocultar Navegador</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QWidget" name="">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>111</width>
             <height>41</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Data de Fechamento</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="date_execucao">
              <property name="wrapping">
               <bool>false</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::UpDownArrows</enum>
              </property>
              <property name="accelerated">
               <bool>false</bool>
              </property>
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
              <property name="date">
               <date>
                <year>2025</year>
                <month>1</month>
                <day>10</day>
               </date>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>480</x>
            <y>10</y>
            <width>181</width>
            <height>91</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 87 italic 16pt &quot;Segoe UI&quot;;
color: rgb(255, 0, 0);</string>
          </property>
          <property name="text">
           <string>Execute com consciência, lebresse, não existe retorno. </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>480</x>
            <y>100</y>
            <width>201</width>
            <height>51</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(170, 255, 0);</string>
          </property>
          <property name="text">
           <string>Executar</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QListView" name="listView"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>714</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCadastro">
    <property name="title">
     <string>Cadastro</string>
    </property>
    <addaction name="actionRespostas"/>
   </widget>
   <widget class="QMenu" name="menuSistema">
    <property name="title">
     <string>Sistema</string>
    </property>
    <addaction name="actionConfigura_es"/>
   </widget>
   <widget class="QMenu" name="menuExecus_o">
    <property name="title">
     <string>Execusão</string>
    </property>
    <addaction name="actionLog_de_Execus_o"/>
   </widget>
   <addaction name="menuSistema"/>
   <addaction name="menuCadastro"/>
   <addaction name="menuExecus_o"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRespostas">
   <property name="text">
    <string>Respostas</string>
   </property>
  </action>
  <action name="actionSistemas">
   <property name="text">
    <string>Sistemas</string>
   </property>
  </action>
  <action name="actionConfigura_es">
   <property name="text">
    <string>Configurações</string>
   </property>
  </action>
  <action name="actionLog_de_Execus_o">
   <property name="text">
    <string>Log de Execusão</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
